pages/about/groups/task-forces/conformance-testing.md

```diff
@@ -32,86 +30,63 @@ Some information on this page is also shown on, and may be more current in, the
+* If you’re aware of an issue with any of the published ACT Rules resources, raise a new issue.
+* Comment on or propose solutions for open ACT Rules issues.
 
-### Tools and Information
+### Become a participant in the task force
 
-- [ACT TF Work Statement](/about/groups/task-forces/conformance-testing/work-statement/)
-- [Wiki main page](https://www.w3.org/WAI/GL/task-forces/conformance-testing/wiki/)
-    - [ACT Overview](https://www.w3.org/WAI/GL/task-forces/conformance-testing/wiki/ACT_Overview_-_What_is_ACT)
-    - [Meeting minutes](https://www.w3.org/WAI/GL/task-forces/conformance-testing/wiki/Meeting_Minutes) (also see [automatically collected minutes](https://www.w3.org/WAI/GL/task-forces/conformance-testing/minutes))
-    - [Availability for ACT TF Teleconferences](https://www.w3.org/2002/09/wbs/93339/availability/)
-- [Mailing list archive](http://lists.w3.org/Archives/Public/public-wcag-act/)
-- [Teleconference logistics](https://www.w3.org/WAI/GL/task-forces/conformance-testing/wiki/Teleconference_Logistics)
-    - [Zoom and IRC setup](https://www.w3.org/WAI/GL/task-forces/conformance-testing/wiki/Zoom_and_IRC_Setup)
-    - [Scribing instructions](https://www.w3.org/WAI/GL/task-forces/conformance-testing/wiki/Scribing_Instructions)
-    - [Scribe rotation list](https://www.w3.org/WAI/GL/task-forces/conformance-testing/wiki/Scribe_Rotation_List)
-- [Face-to-face meetings](https://www.w3.org/WAI/GL/task-forces/conformance-testing/wiki/Face-to-Face_Meetings)
-- GitHub repositories
-    - [ACT Spec (main)](https://github.com/w3c/wcag-act/)
-    - [ACT Rules](https://github.com/w3c/wcag-act-rules/)
-- [Web-based surveys (WBS)](https://www.w3.org/2002/09/wbs/93339/)
+Joining the ACT Task Force enables you to participate fully in the development of the work and influence the deliverables. You and your organization will also be listed as contributors, where appropriate.
 
-## Current Work
+Being a participant involves commitment to support the work of the task force in the following ways:
 
-See the [wiki for current planning and draft documents](https://www.w3.org/WAI/GL/task-forces/conformance-testing/wiki/).
+* Engage actively in the work using the task force’s communication channels.
+* Keep up with weekly tasks and the progress of the work — for example, via the minutes of past meetings, mailing list discussions, and GitHub issue comments.
+* Give your input promptly, when it’s needed.
 
-## Publications
+To become a participant in the ACT Task Force, you must first be a member of the AG Working Group. See [Instructions for joining the Accessibility Guidelines Working Group](https://www.w3.org/groups/wg/ag/instructions/).
 
-**Accessibility Conformance Testing (ACT) Rules Format 1.0**
+Once you are a member of the AG Working Group, [email the ACT Task Force chairs](https://www.w3.org/groups/tf/wcag-act/participants/#chairs) to let them know you’d like to join the task force. 
```

Comment:
Email should be to Team Contact: https://www.w3.org/groups/tf/wcag-act/participants/#staff